terraform/gitops/pm4ml/pm4ml.tf

```diff
@@ -84,6 +84,9 @@ module "generate_pm4ml_files" {
     role_assign_svc_secret_name                     = join("$", ["", "{${replace("${var.role_assign_svc_secret_prefix}${each.key}", "-", "_")}}"])
     role_assign_svc_user                            = var.role_assign_svc_user
     pm4ml_reserve_notification                      = each.value.pm4ml_reserve_notification
+    pm4ml_core_connector_image_repository           = var.pm4ml_core_connector_image_repository
+    pm4ml_core_connector_image_tag                  = var.pm4ml_core_connector_image_tag
+    pm4ml_core_connector_env                        = var.pm4ml_core_connector_env
```

Comment:
try using the app_var_map variable like this:     
admin_portal_chart_version                      = try(var.app_var_map.admin_portal_chart_version, var.admin_portal_chart_version)

you can try adding variables in the pm4ml-vars.yaml file like:
pm4ml_core_connector_image_repository: https://image_repo
pm4ml_core_connector_image_tag: v12.0.0
core_connector_env_vars:
  BACKEND_ENDPOINT: "http://localhost:3002"
   MLCONN_OUTBOUND_ENDPOINT:  "http://localhost:4001"
   
@vijayg10 and @sijo5722-2021  please comment here

Reply:
Refactored